tools/ubuntu-setup/scala.sh

```diff
@@ -2,7 +2,7 @@
 set -e
 set -x
 
-wget www.scala-lang.org/files/archive/scala-2.11.6.deb -O /tmp/scala-2.11.6.deb
-sudo dpkg -i /tmp/scala-2.11.6.deb
+wget www.scala-lang.org/files/archive/scala-2.11.8.deb -O /tmp/scala-2.11.8.deb
+sudo dpkg -i /tmp/scala-2.11.8.deb
```

Comment:
I think we can throw this out of the setup. Gradle should grab the scala compiler dynamically.